pkg/conversion/cli.go

```diff
@@ -192,8 +193,11 @@ func consumeState(state []byte, tr resource.Terraformed) (managed.ConnectionDeta
 	}
 
 	conn := managed.ConnectionDetails{}
-	if err = JSParser.Unmarshal(st.GetSensitiveAttributes(), &conn); err != nil {
-		return nil, errors.Wrap(err, "cannot parse connection details")
+	sensitive := st.GetSensitiveAttributes()
+	if jsoniter.Get(sensitive, '*').Size() > 0 {
```

Comment:
```suggestion
	if len(sensitive) > 0 {
```

Wouldn't checking the length work?

Reply:
Nope, when there is not value we get `[]byte("[]")` here which would return len = 2. 
We need to consider this as empty.